vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -0,0 +1,257 @@
+```
+
+  </Step>
+  <Step>
+
+Download the `loft-images.txt` file and the required scripts `download-images.sh` and `push-images.sh`, then make them executable:
+
+```bash title="Download and prepare scripts"
+wget https://github.com/loft-sh/vcluster/releases/download/v${VERSION}/vcluster-images.txt
+wget https://github.com/loft-sh/vcluster/releases/download/v${VERSION}/download-images.sh
+wget https://github.com/loft-sh/vcluster/releases/download/v${VERSION}/push-images.sh
+
+chmod +x ./download-images.sh
+chmod +x ./push-images.sh
+```
+
+  </Step>
+  <Step>
+
+Run `download-images.sh` to download all images locally:
+
+```bash title="Download images"
+./download-images.sh --image-list loft-images.txt
+```
+
+  </Step>
+  <Step>
+
+Run `push-images.sh` to push all downloaded images to your private registry:
+
+```bash title="Push images to private registry"
+./push-images.sh --registry ${REGISTRY}
+```
+
+    </Step>
+    <Step>
+
+Edit your existing `vcluster.yaml` file or create a new one with the following content:
+
+```yaml title="vcluster-platform.yaml"
+image: ${REGISTRY}/ghcr.io/loft-sh/vcluster:${VERSION} # Replace ${REGISTRY} and ${VERSION}
+env:
+  DEFAULT_IMAGE_REGISTRY: ${REGISTRY} # Replace ${REGISTRY}
+```
+
+:::info Example of a fully formed registry path
+If your `REGISTRY` is set to `ecr.io/myteam`, a fully formed registry path might look like:
+`ecr.io/myteam/ghcr.io/loft-sh/vcluster:0.21.0`
+:::
+
+    </Step>
+
+</Flow>
+
+:::info Note on alpine image replacement
+The `alpine` image may be replaced with any similar image with the following `vcluster.yaml` configuration:
+
+```yaml title="vcluster.yaml configuration for custom image"
+sync:
+  toHost:
+    pods:
+      rewriteHosts:
+        initContainer:
+          image: your-registry/your-image:1.0.0
+```
+
+The image that is used to replace the default image must be able to run the following command:
+
+```yaml
+Command: []string{"sh"},
+Args:    []string{"-c", "sed -E -e 's/^(\\d+.\\d+.\\d+.\\d+\\s+)" + fromHost + "$/\\1 " + toHostnameFQDN + " " + toHostname + "/' /etc/hosts > /hosts/hosts"}
+```
+:::
+
+## Configure vCluster for air-gapped install
+
+The `vcluster.yaml` file holds your vCluster configuration and allows overriding the vCluster Helm chart default values.
+
+The following are some default values that may need to be overridden for a vCluster Pro air-gapped install:
+
+### Set default image registry
+
+Use `defaultImageRegistry` to configure a private image registry.
+
+```yaml title="Default Image Registry configuration"
+controlPlane:
+  advanced:
+    defaultImageRegistry: my-private-registry:5000/vcluster/
+```
+
+vCluster prepends the image registry to all images used by vCluster, such as syncer and Kubernetes. For example, `registry.k8s.io/kube-apiserver:v1.30.2` becomes `my-private-registry:5000/vcluster/kube-apiserver:v1.30.2`.
+
+### Create image pull secret
```

Comment:
I am unclear if this is necessary

Reply:
It may be - it depends on if their container registry requires it or not, and if it needs to be configured at the pod level. So maybe some additional context is in order?